core/modules/dashboard/includes/block.welcome.inc

```diff
@@ -93,15 +117,18 @@ class DashboardWelcomeBlock extends Block {
       );
     }
 
-    $more_actions_links = array();
-    $more_actions_links['admin/modules'] = t('Turn existing features on or off');
-    $more_actions_links['admin/modules/install'] = t('Add more functionality');
-    $more_actions_links['https://backdropcms.org/user-guide'] = t('Read the online user guide');
-    $more_actions_links['https://forum.backdropcms.org'] = t('Visit the Backdrop CMS Forum');
-    $more_actions_links = $this->filterAvailableLinks($more_actions_links);
+    // Add the third column of links.
+    $links_col3 = array();
+    $links_col3['admin/modules'] = t('Turn existing features on or off');
```

Comment:
We don't use the word "features" in core. This should either be "functionality" or (I would prefer) "modules".
```suggestion
    $links_col3['admin/modules'] = t('Turn existing modules on or off');
```

Reply:
I think that we should go with functionality, as Drupal/Backdrop novices may not know what a module is. Once they click that link and land on the "Modules" page, they will make the cor=relation between the two.